responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -18,5 +18,5 @@ dependencies:
     - plotly==5.6.0
     - kaleido==0.2.1
     - mltable==1.0.0
-    - https://publictestdatasets.blob.core.windows.net/packages/pypi/responsibleai_tabular_automl/responsibleai_tabular_automl-0.0.0-py3-none-any.whl
+    - responsibleai-tabular-automl
```

Comment:
Would you please pin the version for more consistent builds of the Docker image? If you'd like to avoid manually incrementing the version, you can use the {{latest-pypi-version}} template tag, as mentioned here: https://github.com/Azure/azureml-assets/wiki/Environments#template-tags

Reply:
done. Thanks!